v23.1/cockroach-debug-encryption-decrypt.md

```diff
@@ -0,0 +1,63 @@
+---
+title: cockroach debug encryption-decrypt
+summary: Learn the commands for collecting debug information from all nodes in your cluster.
+toc: true
+docs_area: reference.cli
+---
+
+{{site.data.alerts.callout_danger}}
+We strongly recommend only using `cockroach debug encryption-decrypt` when working directly with the [Cockroach Labs support team](support-resources.html).
+{{site.data.alerts.end}}
+
+The [`cockroach debug encryption-decrypt`](cockroach-commands.html) command connects to your cluster and decrypts store files that are [encrypted at rest](encryption.html) so that they can be examined to add more context to logs gathered from the [`cockroach debug zip`](cockroach-debug-zip.html) command. Unless they are decrypted, Cockroach Labs cannot examine store files that are encrypted at rest.
+
+## Synopsis
+
+~~~ shell
+$ cockroach debug encryption-decrypt --enterprise-encryption={encryption_spec} --url=<connection string> {flags} {store_directory} {input_file} {output_file}
+~~~
+
+Replace:
+
+- `{encryption_spec}`: The cluster's [encryption details](encryption.html#starting-a-node-with-encryption).
+- `{store_directory}`: The directory where the node stores data files.
+- `{input_file}`: The name of an encrypted store file.
+- `{output_file}`: A name for the decrypted store file. If not specified, the command will output the decrypted contents to
+`STDOUT`.
+
+After running the command, you can send the requested decrypted file to Cockroach Labs.
+
+## Subcommands
+
+{% include {{ page.version.version }}/misc/debug-subcommands.md %}
+
+## Flags
+
+The `debug encryption-decrypt` subcommand supports the following [general-use](#general) and [client connection](#client-connection) flags.
```

Comment:
Might need to rework this section. The flags don't look right.

Here are the supported flags (from running `--help`):

```
--enterprise-encryption StoreEncryptionSpec
-h, --help

Global Flags:
--log <string>
-log='file-defaults: {dir: ...}'.
--log-config-file <file>
--log-config-vars strings
--log-dir <string>
--logtostderr <severity>[=DEFAULT]
--redactable-logs
--vmodule moduleSpec
```

Not sure if those are even worth mentioning, given they are all related to logging, and not the functioning of the command itself.

Reply:
Probably not worth mentioning since we don't document these log options for other debug subcommands. What is `--vmodule` about? Something they might change without support asking them to?

Reply:
> What is --vmodule about? Something they might change without support asking them to?

It's for controlling the log verbosity. We'd only ever ask them to alter this if we were debugging an issue in the command itself. I think we can omit.